app.json

```diff
@@ -1,11 +1,11 @@
 {
   "expo": {
-    "name": "carry-through-covid-app",
-    "slug": "carry-through-covid-app",
+    "name": "carry-app",
```

Comment:
I think this is the app name that I end up seeing underneath the app icon on iPhone. Would be good to change to "Carry" before next release.

Reply:
Ahh v good pickup. Fixed.